src/libs/SearchUIUtils.ts

```diff
@@ -1388,7 +1388,7 @@ function createAndOpenSearchTransactionThread(
     shouldNavigate = true,
 ) {
     const iouReportAction = getIOUActionForReportID(item.reportID, item.transactionID);
-    const moneyRequestReportActionID = item.moneyRequestReportActionID !== '0' ? item.moneyRequestReportActionID : undefined;
+    const moneyRequestReportActionID = item.reportAction?.reportActionID ?? undefined;
```

Comment:
With this change `'0' ?? undefined` will return `'0'` instead of undefined(what we have according to the current implementation)